purchase_force_invoiced/model/purchase.py

```diff
@@ -23,15 +23,3 @@ def _get_invoiced(self):
             lambda po: po.force_invoiced and po.invoice_status == "to invoice"
         ):
             order.invoice_status = "invoiced"
-
-
-class PurchaseOrderLine(models.Model):
-    _inherit = "purchase.order.line"
-
-    def _prepare_account_move_line(self, line):
```

Comment:
Need to remove this function? It does make sense to have it, isn't it?

Reply:
This function is removed due to in 14.0 version, the field doesn't work with this function